File: Matrix/Java/Multiply_Matrices.java
/*Complete the function below*/
class GfG { 
    public static void multiply(int A[][], int B[][], int C[][], int N) {
        //add code here.
        for (int i=0; i<N; i++) {
            for (int j=0; j<N; j++) {
                C[i][j] = 0;
                for (int k=0; k<N; k++) {
                    C[i][j] += A[i][k] * B[k][j];
                }
            }
        }
    }
}

File: Matrix/Python/Multiply_Matrices.py
# Your task is to complete this function
# function should store the multiplication in C
def multiply(A, B, C, n):
    # Code here
    for i in range(len(A)):
        for j in range(len(B)):
            for k in range(len(B[j])):
                C[i][j] += A[i][k]*B[k][j]
    return C
